packages/subgraph/tests/helpers/method-classes.ts

```diff
@@ -82,6 +88,8 @@ import {
   createTokenCalls
 } from '../utils';
 
+/* eslint-disable  @typescript-eslint/no-unused-vars */
```

Comment:
Any particular for this or you forgot?

Reply:
yes since these classes is not exported,
i added this to silence the lint warning in this file only